src/components/calcite-stepper-item/calcite-stepper-item.tsx

```diff
@@ -159,7 +159,7 @@ export class CalciteStepperItem {
     }
   }
 
-  @Listen("calciteStepperItemHasChanged", { target: "parent" })
+  @Listen("calciteStepperItemChanged", { target: "parent" })
```

Comment:
I think this should be `calciteStepperItemChange` everywhere so it's present-tense

Reply:
Oh dang!
`calciteStepperWouldHaveBeenChanged`?
jk 
will do

Reply:
Thanks @paulcpederson. I did the thing.